go/src/koding/kites/config/cache.go

```diff
@@ -83,7 +83,7 @@ func newBoltDB(o *CacheOptions) (*bolt.DB, error) {
 	return bolt.Open(o.File, 0644, o.BoltDB)
 }
 
-func KodingHome() string {
+func KodingConfigHome() string {
```

Comment:
Why do we need to rename it? The env is `KODING_HOME` so we can still treat it as KodingHome. Koding home directory stores also kd / klient data and cache, so it's not solely a config dir.